server/conf/application.conf

```diff
@@ -33,51 +33,116 @@ addressIndex {
+        lpiLocalityBoost=0.5
+        lpiLocalityBoost=${?ONS_AI_API_QUERY_TOWN_NAME_LPI_LOCALITY_BOOST}
+        pafDoubleDependentLocalityBoost=0.2
+        pafDoubleDependentLocalityBoost=${?ONS_AI_API_QUERY_TOWN_NAME_PAF_DOUBLE_DEPENDENT_LOCALITY_BOOST}
+        pafWelshDoubleDependentLocalityBoost=0.2
+        pafWelshDoubleDependentLocalityBoost=${?ONS_AI_API_QUERY_TOWN_NAME_PAF_WELSH_DOUBLE_DEPENDENT_LOCALITY_BOOST}
+      }
+      postcode {
+        pafPostcodeBoost=1.0
+        pafPostcodeBoost=${?ONS_AI_API_QUERY_POSTCODE_PAF_POSTCODE_BOOST}
+        lpiPostcodeLocatorBoost=1.0
+        lpiPostcodeLocatorBoost=${?ONS_AI_API_QUERY_POSTCODE_LPI_POSTCODELOCATOR_BOOST}
+        postcodeOutBoost=0.8
+        postcodeOutBoost=${?ONS_AI_API_QUERY_POSTCODE_PAF_OUTCODE_BOOST}
+        postcodeInBoost=0.3
+        postcodeInBoost=${?ONS_AI_API_QUERY_POSTCODE_PAF_INCODE_BOOST}
+      }
+      organisationName {
+        pafOrganisationNameBoost=1.0
+        pafOrganisationNameBoost=${?ONS_AI_API_QUERY_ORGANISATION_NAME_PAF_ORIGANISATION_NAME_BOOST}
+        lpiOrganisationBoost=1.0
+        lpiOrganisationBoost=${?ONS_AI_API_QUERY_ORGANISATION_NAME_LPI_ORIGANISATION_BOOST}
+        lpiPaoTextBoost=1.0
+        lpiPaoTextBoost=${?ONS_AI_API_QUERY_ORGANISATION_NAME_LPI_PAO_TEXT_BOOST}
+        lpiLegalNameBoost=1.0
+        lpiLegalNameBoost=${?ONS_AI_API_QUERY_ORGANISATION_NAME_LPI_LEGALNAME_BOOST}
+        lpiSaoTextBoost=0.5
+        lpiSaoTextBoost=${?ONS_AI_API_QUERY_ORGANISATION_NAME_LPI_SAO_TEXT_BOOST}
+      }
+      departmentName {
+        pafDepartmentNameBoost=1.0
+        pafDepartmentNameBoost=${?ONS_AI_API_QUERY_DEPARTMENT_NAME_PAF_DEPARTMENT_NAME_BOOST}
+        lpiLegalNameBoost=0.5
+        lpiLegalNameBoost=${?ONS_AI_API_QUERY_DEPARTMENT_NAME_LPI_LEGAL_NAME_BOOST}
+      }
+      locality {
+        pafDependentLocalityBoost=1.0
+        pafDependentLocalityBoost=${?ONS_AI_API_QUERY_LOCALITY_PAF_DEPENDENT_LOCALITY_BOOST}
+        pafWelshDependentLocalityBoost=1.0
+        pafWelshDependentLocalityBoost=${?ONS_AI_API_QUERY_LOCALITY_PAF_WELSH_DEPENDENT_LOCALITY_BOOST}
+        lpiLocalityBoost=1.0
+        lpiLocalityBoost=${?ONS_AI_API_QUERY_LOCALITY_LPI_LOCALITY_BOOST}
+        pafDoubleDependentLocalityBoost=0.5
+        pafDoubleDependentLocalityBoost=${?ONS_AI_API_QUERY_LOCALITY_PAF_DOUBLE_DEPENDENT_LOCALITY_BOOST}
+        pafWelshDoubleDependentLocalityBoost=0.5
+        pafWelshDoubleDependentLocalityBoost=${?ONS_AI_API_QUERY_LOCALITY_PAF_WELSH_DOUBLE_DEPENDENT_LOCALITY_BOOST}
+      }
+      pafAllBoost=0.5
+      pafAllBoost=${?ONS_AI_API_QUERY_PAF_ALL_BOOST}
+      nagAllBoost=0.5
+      nagAllBoost=${?ONS_AI_API_QUERY_NAG_ALL_BOOST}
+      defaultBoost=1.0
+      defaultBoost=${?ONS_AI_API_QUERY_DEFAULT_BOOST}
       minimumShouldMatch="45%"
       minimumShouldMatch=${?ONS_AI_API_QUERY_MINIMUM_SHOULD_MATCH}
     }
```

Comment:
We don't know the limit with the new query yet. We'll update it after the first tests